requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django~=3.2.10
```

Comment:
Add whitespaces for readability
```suggestion
Django ~= 3.2.10
```

Reply:
https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/17e03fa76b7c9c897687aac54fc8ccf334c37074

Reply:
Marking this as resolved.